osu.Game/GameModes/Menu/ButtonSystem.cs

```diff
@@ -285,37 +290,35 @@ public override void Load()
             {
                 base.Load();
 
-                Add(logoBounceContainer = new AutoSizeContainer());
-
-                logo = new Sprite(Game.Textures.Get(@"menu-osu"))
-                {
-                    Anchor = Anchor.Centre,
-                    Origin = Anchor.Centre,
-                };
-
-                logoBounceContainer.Add(logo);
-
                 Sprite ripple = new Sprite(Game.Textures.Get(@"menu-osu"))
                 {
                     Anchor = Anchor.Centre,
                     Origin = Anchor.Centre,
                     Alpha = 0.4f
                 };
-
-                logoBounceContainer.Add(ripple);
-
                 ripple.ScaleTo(1.1f, 500);
                 ripple.FadeOut(500);
                 ripple.Loop(300);
 
-                logoBounceContainer.Add(vis = new MenuVisualisation()
+                Add(logoBounceContainer = new AutoSizeContainer()

```

Comment:
This leads me to a (simple) question, this (and probably other) only has one child (the child has many children but there's only one child here), I'm not sure whether you want:
* ```Add()``` to avoid creating an array of 1 object (though in the future it may well expand)
or
* ```Children = new Drawable[] {}``` for Consistency